### PR TITLE
Add parameters for sql container and gremlin graph commands

### PR DESCRIPTION
**Pull request by bizarreWizard**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by bizarreWizard** [comment]
Can anyone review the PR?

**Comment by yonzhan** [comment]
@Juliehzl  please help review and comment.

File: src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py [hunk 1]
```diff
@@ -431,7 +469,11 @@ def cli_cosmosdb_gremlin_graph_update(client,
     gremlin_graph_resource.unique_key_policy = gremlin_graph.unique_key_policy
     gremlin_graph_resource.conflict_resolution_policy = gremlin_graph.conflict_resolution_policy
 
-    if _populate_gremlin_graph_definition(gremlin_graph_resource, None, default_ttl, indexing_policy):
+    if _populate_gremlin_graph_definition(gremlin_graph_resource,
+                                          None,
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py]: What does `None` parameter mean here?

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -15,6 +15,10 @@ Release History
 
 * az network private-dns link vnet create/update: Fixes #9851. Support cross-tenant virtual network linking.
 
+**CosmosDB**
+* Add parameter for sql container create and update command
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst]: Can you describe your command change in detail? Besides sql containter, you also add a parameter for gremlin and declare the parameter name here.

In addition, you need to rearrange your release note to make all cli modules in alphabetical order here.

**Comment by Juliehzl** [comment]
Can you add test scenario for your new parameter?

**Comment by yonzhan** [comment]
@bizarreWizard please look into Juliehzl's comment.

File: src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py [hunk 3]
```diff
@@ -104,6 +110,8 @@ def load_arguments(self, _):
         c.argument('partition_key_path', options_list=['--partition-key-path', '-p'], help='Partition Key Path, e.g., \'/address/zipcode\'')
         c.argument('default_ttl', options_list=['--ttl'], type=int, help='Default TTL. If the value is missing or set to "-1", items don’t expire. If the value is set to "n", items will expire "n" seconds after last modified time.')
         c.argument('indexing_policy', options_list=['--idx'], type=shell_safe_json_parse, completer=FilesCompleter(), help='Indexing Policy, you can enter it as a string or as a file, e.g., --idx @policy-file.json or ' + SQL_GREMLIN_INDEXING_POLICY_EXAMPLE)
+        c.argument('unique_key_policy', options_list=['--unique'], type=shell_safe_json_parse, completer=FilesCompleter(), help='Unique Key Policy, you can enter it as a string or as a file, e.g., --unique @policy-file.json or ' + SQL_UNIQUE_KEY_POLICY_EXAMPLE)
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py]: There are two problems here: 1. --unique is a little strange, can you change to another one? 2. Can you also provide full argument name in your options_list like `partition_key_path`?

The same for conflict_resolution_policy.

**Comment by yonzhan** [comment]
@bizarreWizard There still needs some improvement for your command and it seemed that you are on vacation these days. We will merge this PR in next Sprint.

**Comment by Juliehzl** [comment]
Hi @bizarreWizard, please update history.rst as what we have discussed. Note to add [BREAKING CHANGE] for required parameter change.

**Pull request merged**